### ISSUE TITLE
Undefined index/caught TypeError after updating to latest develop

### ISSUE THREAD

**Issue by Quix0r** [friendica/friendica#7978]
With the current `develop` code, I got these 2 errors, it did never happen with the code from October:
````
Notice: Undefined index: prvKey in /var/www/.../src/App/Authentication.php on line 355

Fatal error: Uncaught TypeError: Argument 3 passed to Friendica\Model\User\Cookie::set() must
be of the type string, null given, called in /var/www/.../src/App/Authentication.php on line 355 and
defined in /var/www/.../src/Model/User/Cookie.php:78 Stack trace: #0 /var/www/.../src/App/Authentication.php(355): Friendica\Model\User\Cookie->set(1, '$2y$10$RB4l5JBC...', NULL) 
#1 /var/www/.../src/App/Authentication.php(267): Friendica\App\Authentication->setForUser(Object(Friendica\App), Array, true, true) 
#2 /var/www/.../src/Module/Login.php(62): Friendica\App\Authentication->withPassword(Object(Friendica\App), 'roland', 'xxx', true)
#3 [internal function]: Friendica\Module\Login::post(Array)
#4 /var/www/.../src/App/Module.php(260): call_user_func(Array, Array)
#5 /var/www/.../src/App.php(785): Frie in /var/www/.../src/Model/User/Cookie.php on line 78
````
PS: Nasty! The backtrace contains my password ... :-(

**Edit:** It only happens when you **delete cookies before** you browse to your Friendica instance.

**Comment by annando** [comment]
"develop" doesn't get much love at the moment, since we are working on the "rc".

**Comment by Quix0r** [comment]
Then I'm doomed or what? Can't this be quickly fixed?

**Comment by annando** [comment]
You can switch to the "rc" branch. This is the fastest way.

**Comment by Quix0r** [comment]
I always stick with the `develop` in the past. Please, before leaving something damaged behind, make at least sure that login is possible!

**Comment by MrPetovan** [comment]
This doesn't look like the current develop, the `Authentication.php` file has been stripped down to only 96 lines. Please make sure you fetched and pulled `develop` correctly first.

**Comment by Quix0r** [comment]
A quick `git diff develop..upstream/develop` returns nothing, means my local `develop` is in sync with `upstream` (main Friendica, means this repository).

**Comment by Quix0r** [comment]
Last item:
````
commit 6e4a428c7394ecb82675ce223a87daf721e6894b (HEAD -> develop, upstream/develop, server/develop, origin/develop)
Merge: d0b20fb49 b82294826
Author: Hypolite Petovan <hypolite@mrpetovan.com>
Date:   Sat Dec 14 09:53:40 2019 -0500

    Merge pull request #7907 from nupplaphil/task/reduce_app_deps
    
    Cleanup Session/Authentication
````

**Comment by MrPetovan** [comment]
Can you please try to manually change `prvKey` to `prvkey` on line 355 of `src/App/Authentication.php`?

**Comment by Quix0r** [comment]
I found 3 ocurrances and have fixed all 3 to `prvkey`.

**Comment by MrPetovan** [comment]
Did it fix the issue?

**Comment by Quix0r** [comment]
Yes, thanks again for pointing me into the right direction. It was a long and stressful week for me. Hope you can enjoy your weekend all @annando @MrPetovan . :-)

**Comment by MrPetovan** [comment]
Fixed by #7979
